packages/react/src/utils/styles.tsx

```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 
+export const DISABLE_SCROLLBAR_CLASS_NAME = 'base-ui-disable-scrollbar';
```

Comment:
This is already available as `styleDisableScrollbar.className`, why expose it in a different way? Also are we sure we want to expose this publicly now? IIUC, this is just for the experiment page, we should avoid adding public exports for that, they become subject to semver if we do.